### PR TITLE
BNPL Send total amount to passport web app [25581]

### PR DESCRIPTION
**Pull request by lqgioan**
## Description
- Passport `OrderInfo`(orderId, orderAmount) to the passport app.
- Added `availableTerms` to `ProductDetail` object.

## Motivation & Context
Ticket: https://app.shortcut.com/credify/story/24892/passport-web-sdk-map-bnpl-product-into-passport-app

## How has this been tested?
Test BNPL and it should show about `interest rate` info

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: Credify/Credify/Objects/CommonModel.swift [hunk 1]
```diff
@@ -238,10 +238,12 @@ public class JSONCodingKey: CodingKey {
...
 
-internal class OrderInfo : Codable {
+public class OrderInfo : Codable {
     let orderId: String
+    let orderAmount: Double
     
-    init(orderId: String) {
+    public init(orderId: String, orderAmount: Double) {
         self.orderId = orderId
+        self.orderAmount = orderAmount
```
Comment (ngo275) [inline Credify/Credify/Objects/CommonModel.swift:247]: Not FiatCurrency?

Comment (lqgioan) [reply Credify/Credify/Objects/CommonModel.swift:247]: @ngo275 yes. It's price. E.g: 150000

Comment (ngo275) [reply Credify/Credify/Objects/CommonModel.swift:247]: What about the currency information? We should keep the currency information too

Comment (lqgioan) [reply Credify/Credify/Objects/CommonModel.swift:247]: @ngo275 I update my code. And updated the Passport web app and Web SDK too

**Review by ngo275 — APPROVED** [review]

**Pull request merged**